### PR TITLE
Disable feed file generation. 

### PR DESCRIPTION
**Pull request by budzanowski**
This is temporary. There is a chance that this is permanent if we will switch all the sync operations to the Batch API.

### CODE REVIEW

**Comment by rcstr** [comment]
Updated base branch to `base-2.3.6`

File: includes/Products/Feed.php [hunk 1]
```diff
@@ -53,8 +53,8 @@ public function __construct() {
 	 */
 	private function add_hooks() {
 
-		// schedule the recurring feed generation
-		add_action( 'init', [ $this, 'schedule_feed_generation' ] );
+		// schedule the recurring feed generation -  temporarily disabled.
+		// add_action( 'init', [ $this, 'schedule_feed_generation' ] );
```
Comment (rcstr) [inline includes/Products/Feed.php:57]: I'd say we remove this line, we can always revert if we need this again.

**Review by rcstr — APPROVED** [review]
@budzanowski LGTM - I was able to test the behavior by running:
``` shell
> wp shell
> as_unschedule_all_actions( facebook_for_woocommerce()-> get_product_feed_handler()::GENERATE_FEED_ACTION ); 
```

this disabled the scheduled job as expected.

Left an observation

**Comment by haszari** [comment]
@budzanowski  Can you open an issue proposing this change? As you're aware, the feed file may still be used by some stores so we'll need to take care here and ensure the change is clearly signalled (issue, PR implementing change, release notes/docs etc). (I'm assuming this PR is blocked on discussing the possible impact of disabling feed file generation - added `blocked`.)

I suspect you're aware of these considerations - just a reminder that we need an issue to work through the implications of this change.

**Comment by haszari** [comment]
Closing - as discussed elsewhere, the feed file is an essential part of sync, and we're currently focused on getting it working well. @budzanowski let me know or comment if this PR includes changes that you intend to ship - we can reopen or open another PR if needed.

**Pull request closed without merging**